### ISSUE TITLE
permanent errors from ossn_add_tokens_to_url($url)

### ISSUE THREAD

**Issue by githubertus** [opensource-socialnetwork/opensource-socialnetwork#127]
Navigating around in my testsite, system/templates/output/url.php is called very often,
issuing a ossn_add_tokens_to_url($url)
with $url = 'http://ossn/action/user/logout'

On the other hand, within the function ossn_add_tokens_to_url(), the line
$params = parse_url($url); gives just
![security-parms](https://cloud.githubusercontent.com/assets/9904364/5411860/7ba1e58e-8206-11e4-9ba3-19780e2107c2.png)
In other words: $params has NO index named 'query' - leading to the error later on line
parse_str($params['query'],  $query);
...................
ossn 2.0-dev latest (local testsite)
